src/components/DropdownList/DropdownList.scss

```diff
@@ -0,0 +1,9 @@
+.dropdown {
+  &-item {
+    cursor: pointer;
+
+    &:hover p {
+      color: red !important;
```

Comment:
Avoid using '!important' in CSS. It increases the specificity of a rule and can make debugging more difficult. Try to use more specific selectors or re-arrange your CSS rules.